.travis.yml

```diff
@@ -9,16 +17,16 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=4.2.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
```

Comment:
sorry, I'm not sure we can explicitly drop the support for 4.2 and 4.3 in the same minor branch of this module